collections/_posts/2019-08-22-self-hosting-a-shadowsocks-vpn-with-outline.md

```diff
@@ -0,0 +1,84 @@
+cover: /assets/img/2019-08-22-outline-cover.jpg
+tags:
+  - guides
+  - vpn
+  - shadowsocks
+---
+
+**Outline** is a suite of open-source software developed for journalists to safely access their network and the internet while traveling in countries where their activities may be monitored or censored. Despite this, the Outline platform is ideal for a wide range of users, especially less technical users, and users in censored countries like China who may have little to no knowledge about how VPNs or proxies work. Outline consists of two parts, the **Outline Manager** and **Outline Clients**. The Outline Manager is a tool you can use to easily setup remote Outline Servers on your own machines with very little technical skills. The Outline Clients in turn can connect to Outline Servers you configure, to keep your traffic secured.
+
+Technically, Outline is not a true VPN. Rather it uses an open-source SOCKS5 proxy called Shadowsocks which is "[designed to protect your Internet traffic](https://shadowsocks.org/en/index.html)". The Outline client applications however make use of the VPN capabilities of your operating system to send all your traffic through your Outline Server, with no need to configure each application to use the proxy. Thus for most users and most use-cases, there's no difference in functionality between using a normal VPN and an Outline server.
+
+Shadowsocks has the benefit of being far more lightweight than OpenVPN, and it is much more optimized for mobile devices, as it does not require any keep-alive connections. It has existed since 2012 and it is widely used in China due to its censorship-resistant functionality: It is very difficult or impossible to detect and block Shadowsocks traffic automatically.
+
+Please note that **like any VPN**, Outline/Shadowsocks cannot provide nearly the same degree of anonymity as projects like Tor. The primary use-case of Outline and VPNs in general is to keep your traffic hidden from malicious Internet Service Providers and nation-wide mass surveillance. It's a great solution for protecting your data on public wifi networks, but if you want to stay hidden from attackers targeting *you*, there's better tools for the job elsewhere.
+
+Outline is developed by Jigsaw, which is a subsidiary of Alphabet Inc (Google). It is important to note that Jigsaw or Google cannot see your internet traffic when using Outline, because you will be installing the actual Outline Server on your own machine, not Google's. Outline is completely open source and was audited in [2017](https://s3.amazonaws.com/outline-vpn/static_downloads/ros-report.pdf) by Radically Open Security and in [2018](https://s3.amazonaws.com/outline-vpn/static_downloads/cure53-report.pdf) by Cure53, and both security firms supported Jigsaw's security claims. For more information on the data Jigsaw is able to collect when using Outline, see their [article on data collection](https://support.getoutline.org/s/article/Data-collection).
```

Comment:
Is this correct or should it be this:?

> It is important to note that *neither* Jigsaw nor Google ~~cannot~~ can see...

This is one of my confusions with English